packages/workspace/src/command-line/format.ts

```diff
@@ -61,43 +53,56 @@ export function format(command: 'check' | 'write', args: yargs.Arguments) {
   }
 }
 
-function getPatterns(args: NxArgs & { libsAndApps: boolean; _: string[] }) {
-  const allFilesPattern = [MATCH_ALL_PATTERN];
+function getPatterns(
+  args: NxArgs & { libsAndApps: boolean; _: string[] },
+  supportedExtensions: string[],
+  matchAllPattern: string
+): string[] {
+  const allFilesPattern = [matchAllPattern];
 
   try {
     if (args.all) {
       return allFilesPattern;
     }
 
     if (args.projects && args.projects.length > 0) {
-      return getPatternsFromProjects(args.projects);
+      return getPatternsFromProjects(args.projects, matchAllPattern);
     }
 
     const p = parseFiles(args);
     const patterns = p.files
       .filter((f) => fileExists(f))
-      .filter((f) =>
-        PRETTIER_EXTENSIONS.map((ext) => `.${ext}`).includes(path.extname(f))
-      );
+      .filter((f) => supportedExtensions.includes(path.extname(f)));
 
-    return args.libsAndApps ? getPatternsFromApps(patterns) : patterns;
+    return args.libsAndApps
+      ? getPatternsFromApps(patterns, matchAllPattern)
+      : patterns;
   } catch (e) {
     return allFilesPattern;
   }
 }
 
-function getPatternsFromApps(affectedFiles: string[]): string[] {
+function getPatternsFromApps(
+  affectedFiles: string[],
+  matchAllPattern: string
+): string[] {
   const graph = onlyWorkspaceProjects(createProjectGraph());
   const affectedGraph = filterAffected(
     graph,
     calculateFileChanges(affectedFiles)
   );
-  return getPatternsFromProjects(Object.keys(affectedGraph.nodes));
+  return getPatternsFromProjects(
+    Object.keys(affectedGraph.nodes),
+    matchAllPattern
+  );
 }
 
-function getPatternsFromProjects(projects: string[]) {
+function getPatternsFromProjects(
+  projects: string[],
+  matchAllPattern: string
+): string[] {
   const roots = getProjectRoots(projects);
-  return roots.map((root) => `${root}/${MATCH_ALL_PATTERN}`);
+  return roots.map((root) => `${root}/${matchAllPattern}`);
```

Comment:
I think this can just be the `root`.

Reply:
Yeah, but the same could be said of the `all` scenario, we could have the pattern as `.`, but I prefer to give the supported extensions because we already have them and we avoid some extra processing by Prettier.